.github/workflows/r-cmd-check.yaml

```diff
@@ -0,0 +1,35 @@
+on: [push, pull_request]
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: r-lib/actions/setup-r@master
+      - uses: r-lib/actions/setup-pandoc@master
+      - name: Query dependencies
+        run: |
+          install.packages(c('remotes', 'rcmdcheck', 'covr', 'rmarkdown'))
+          remotes::install_deps(dependencies = TRUE)
+          saveRDS(remotes::dev_package_deps(dependencies = TRUE), "depends.Rds", version = 2)
+        shell: Rscript {0}
+      - name: Cache R packages
+        uses: actions/cache@v1
+        with:
+          path: ${{ env.R_LIBS_USER }}
+          key: macOS-r-3.6-${{ hashFiles('depends.Rds') }}
+          restore-keys: macOS-r-3.6-
+      - name: Check
+        run: rcmdcheck::rcmdcheck(args = "--no-manual", error_on = "error")
+        shell: Rscript {0}
+      - name: yihui gitconfig
+        run: |
+          curl https://xran.yihui.org/.gitconfig -o ~/.gitconfig
+      - name: Test coverage
+        run: covr::codecov()
+        shell: Rscript {0}
+      - name: xran
+        run: |
+          curl https://xran.yihui.org/r-xran | bash
```

Comment:
For this to work, I need to set [an environment variable containing my Github token](https://github.com/yihui/xran/blob/ad12fcfc1e89aea75e7f3f8270b73cca274965cd/r-xran#L12). How do I encrypt that environment variable?